boards/shields/waveshare_at24/doc/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _waveshare_at24:
+
+EEPROM AT24 Waveshare shield
+####################################
```

Comment:
```suggestion
EEPROM AT24 Waveshare shield
############################
```

Reply:
thanks for pointing it out, done in 21dff0f